### PR TITLE
Cleanup jupyter tox dependencies

### PR DESCRIPTION
**Pull request by brs96**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [ ] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [ ] Your contribution is covered by tests

### CODE REVIEW

File: examples/ml-pipelines-node-classification.ipynb [hunk 1]
```diff
@@ -553,124 +490,14 @@
...
-       "0       0             100  [0.07090129256540621, 0.02781562683066478, 0.2...\n",
-       "1       1               5  [0.10248697035946842, 0.04786333496925834, 0.1...\n",
-       "2       2               2  [0.04005138780445344, 0.4716897821336281, 0.04...\n",
-       "3       3               2  [0.03168648772215199, 0.709948522271949, 0.119...\n",
-       "4       4               3  [0.05023837133685246, 0.022112223083451652, 0....\n",
-       "5       5               5  [0.163967792875254, 0.17289763778273642, 0.108...\n",
-       "6       6               6  [0.055509897709602456, 0.11420633716452112, 0....\n",
-       "7       7             100  [0.0357353343079603, 0.06031988891476322, 0.05...\n",
-       "8       8             100  [0.07296843717126494, 0.09662250045306757, 0.2...\n",
-       "9       9               4  [0.056177601249482755, 0.16500456321738488, 0...."
```
Comment (Mats-SX) [inline examples/ml-pipelines-node-classification.ipynb:666]: I believe @nvitucci argued to keep some of the cell outputs for documentational purposes. Personally, I'm neutral.

**Review by Mats-SX — APPROVED** [review]
I'm happy to approve the PR, but I just wanna check with @nvitucci about clearing all the cell outputs, so that we do not end up ping-ponging on that.

File: examples/ml-pipelines-node-classification.ipynb (hunk 1 above)
Comment (nvitucci) [reply examples/ml-pipelines-node-classification.ipynb:666]: Yes, my suggestions was to keep some output for two reasons:
- documentation purposes (see e.g. the example output on https://neo4j.com/docs/graph-data-science-client/current/tutorials/load-data-via-graph-construction/#_load_the_cora_dataset)
- (potentially) testing purposes, i.e. to make sure that a result stay consistent (as an alternative to explicit `asserts`).

If you can leave this untouched for now, so it keeps reflecting the published docs, we can have a separate discussion on it afterwards.

File: examples/ml-pipelines-node-classification.ipynb [hunk 2]
```diff
@@ -78,7 +78,7 @@
     "NEO4J_USERNAME = \"neo4j\"\n",
     "NEO4J_PASSWORD = \"test\"\n",
     "\n",
-    "gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWORD), database=\"neo4j\")\n",
```
Comment (Mats-SX) [inline examples/ml-pipelines-node-classification.ipynb]: We still need this `database="neo4j"` part

Comment (brs96) [reply examples/ml-pipelines-node-classification.ipynb]: thanks, completely missed it...

**Pull request merged**